pages/items/[itemId].tsx

```diff
@@ -2,21 +2,53 @@ import { GetServerSideProps } from 'next';
 import styled from 'styled-components';
 import React from 'react';
 import { ParsedUrlQuery } from 'querystring';
-import { ConItem } from 'types';
+import { IConItem } from 'types';
 import { getConItem } from 'api';
+import { MenuBar, OptionBox, BuyButton } from 'components/base';
+import { ProductionItem } from 'components/domain';
+import { IcoArrow } from 'public/images';
+import { useOption } from 'hooks';
 
 const ItemsPageWrapper = styled.div``;
 
 interface ItemsPageProps {
   data: {
-    conItem: ConItem;
+    conItem: IConItem;
   };
 }
 
 const ItemsPage = ({ data }: ItemsPageProps) => {
   const { conItem } = data;
+  const { toggle, isActive, isChecked, setIsChecked } = useOption({ initialState: false });
 
```

Comment:
밑에 ProductionItem Props로 들어가는 애들은 전부 conItem을 반복적으로 참조하면서 코드양도 늘어나고 비효율적인 것 같아요

```suggestion
const { imageUrl, name, minSellingPrice, originalPrice, conCategory2, id, options} = conItem;
```

이렇게 구조분해 할당해서 props 넣는게 직관적이고 좋아요